copier/questions/essential.yml

```diff
@@ -0,0 +1,25 @@
+---
+# Essential questions
+package_name:
+  type: str
+  placeholder: my_python_package
+  help: Enter the name of the Python package.
```

Comment:
For the sake of uniformity we need to remove . here

Reply:
I agree